src/test/kotlin/com/teamdev/jxbrowser/gradle/JxBrowserPluginFunctionalTest.kt

```diff
@@ -72,14 +71,26 @@ class JxBrowserPluginFunctionalTest {
             .withArguments("build")
             .build()
 
-        assertEquals(SUCCESS, result.task(":build")!!.outcome)
+        assertThat(result.task(":build")!!.outcome).isEqualTo(SUCCESS)
     }
 
     @Test
     @Throws(IOException::class)
     fun `JxBrowser jars are downloaded correctly`() {
         val version = "7.35.2"
```

Comment:
Can we have it as a constant somewhere, so when we release a new version we can easily update it?